package/upgrade/upgrade_node.sh

```diff
@@ -627,6 +627,15 @@ EOF
   GRUBENV_FILE="/oem/grubcustom"
   chroot $HOST_DIR /bin/bash -c "if ! [ -f ${GRUBENV_FILE} ]; then touch ${GRUBENV_FILE}; fi" 
 
+  multiPathEnabled=$(yq '.os.externalStorageConfig.enabled // false' ${HOST_DIR}/oem/harvester.config)
+  if [ ${multiPathEnabled} == false ]
+  then
+    thirdPartyArgs=$(chroot $HOST_DIR grub2-editenv /oem/grubenv list |grep third_party_kernel_args | awk -F"third_party_kernel_args=" '{print $2}')
+    thirdPartyArgs="${thirdPartyArgs} multipath=off"
```

Comment:
check `multipath=off` is already there to avoid mutli appending it?

Reply:
new commit fixes this.